sheets/README.md

```diff
@@ -0,0 +1,128 @@
+
+## Sheet Basics
+
+The terms "main worksheet" and "form responses worksheet" are used below. Here are some
+screenshots to explain what those terms are referring to:
+
+**Enrollment Code Request worksheet tabs:**
+![Enrollment Code Request worksheet tabs](images/enrollment-code-worksheet-tabs.png)
+
+**Change of Enrollment Request worksheet tabs:**
+![Change of Enrollment Request worksheet tabs](images/change-of-enrollment-worksheet-tabs.png)
+
+Some basic details for reading and interacting with these spreadsheets:
+
+1. One cardinal rule to follow for all of the "request"-type sheets: **Do not edit the main worksheet directly, unless the 
+  columns exist specifically for user input**. Examples of columns that are intended for user input are the Finance 
+  columns in the Change of Enrollment sheets.
+1. If a row in any spreadsheet was successfully processed by our app, there should be a timestamp in a column called 
+  "Date Processed"/"Completed Date"/etc., or a success status in the "Status" column.
+1. If there is an error in some row (e.g.: a typo), there should be a message in the "Errors" column, or a 
+  failure status in the "Status" column. See below for instructions on how to respond to errors in 
+  [request sheets](#handling-request-sheet-errors) and in [assignment sheets](#handling-assignment-sheet-errors).
+
+## Handling Request Sheet Errors
+
+As stated above, errors cannot be fixed by directly updating the main worksheet in these spreadsheets. You have
+two options for addressing errors: editing the submitted form data, or setting the row to "ignored".
+
+#### Editing Rows
+
+**When you'll want to edit:** there is some bad data in a form submission (a typo, a naming conflict, etc).
+
+**How to do it:** Find the row in the relevant "Form Responses" worksheet that matches the row in the main 
+spreadsheet and directly edit the columns there. 
+
+Example scenario: On the Enrollment Code Request spreadsheet, some row has some detail in the Errors column indicating that a
+user doesn't exist due to a misspelled email address. To fix this, you would go to the "Form Responses" worksheet 
+(which holds the the raw response data from the form), find the row that matches the one you saw on the main worksheet
+with the error text, and updated the email column value there. 
+
+Google sends file watch requests to our app if anything in the spreadsheet is edited, so those changes to the Form
+Responses data should be handled automatically.
+
+#### Ignoring Rows 
+
+**Request rows cannot be deleted, only set to "ignored".** There are a couple different reasons for this. 
+ 
+**When you'll want to set a row to ignored:** The request is no longer relevant (for example, the request is an accidental repeat). 
+
+**How to do it:** Find the row in the relevant Form Responses worksheet that matches the
+row on the main worksheet** (check the "Editing Rows" section for details), **and enter `=TRUE` into the "Ignored?" 
```

Comment:
Not sure what you're trying to highlight here with the **'s

Reply:
@briangrossman That's [Markdown](https://www.markdownguide.org/getting-started/) notation. Double-*'s style the text as bold. Have a look at the rendered version: https://github.com/mitodl/mitxpro/blob/1563049a7cf4da92a2b3c7e8933b36a5a7e56059/sheets/README.md#ignoring-rows